src/ucp/wireup/select.c

```diff
@@ -1807,6 +1807,31 @@ ucp_wireup_iface_get_perf_attr_flags(const ucp_worker_iface_t *wiface)
     return perf_attr.flags;
 }
 
+static int
+ucp_wireup_is_rma_bw_needed(ucp_context_h context, unsigned ep_init_flags)
+{
+    /* Always needed for memtype copy endpoints */
+    if (ep_init_flags & UCP_EP_INIT_FLAG_MEM_TYPE) {
+        return 1;
+    }
+
+    /* RMA API is used and multi-lane RMA is enabled */
```

Comment:
comment is confusing, we may use rma_bw lane even with single lane (also you check that max_rma_lanes > 0, not 1)